packages/ui/src/components/Designer/RightSidebar/DetailView/index.tsx

```diff
@@ -176,43 +174,51 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
         }],
         props: { autoComplete: "off" }
       },
-      required: { title: i18n('required'), type: 'boolean', span: 8, hidden: defaultSchema?.readOnly },
+      editable: { title: i18n('editable'), type: 'boolean', span: 8, bind: false, hidden: defaultSchema?.readOnly !== undefined },
+      readOnly: { type: 'boolean', span: 0, hidden: true },
+      required: { title: i18n('required'), type: 'boolean', span: 16, hidden: "{{formData.readOnly}}" },
```

Comment:
I don't think you need `readOnly` in the form since you map it's values to `editable` you should be able to do:
```suggestion
      editable: { title: i18n('editable'), type: 'boolean', span: 8, hidden: defaultSchema?.readOnly !== undefined },
      required: { title: i18n('required'), type: 'boolean', span: 16, hidden: "{{!formData.editable}}" },
```